packages/database/src/migrations/20210507014219-PutWaiviviaToTheRightPlace-modifies-data.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+import { codeToId, insertObject, generateId } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const hierarchyNameToId = async (db, name) => {
+  const record = await db.runSql(`SELECT id FROM entity_hierarchy WHERE name = '${name}';`);
+  return record?.rows[0].id;
+};
+
+const WAIVIVIA_CODE = 'FJ_WAV';
+const BURETA_CODE = 'FJ_CATCH_Bureta';
+const EASTERN_CODE = 'FJ_Eastern';
+const projectCode = 'wish';
+
+exports.up = async function (db) {
+  const WAIVIVIA_ID = await codeToId(db, 'entity', WAIVIVIA_CODE);
+  const BURETA_ID = await codeToId(db, 'entity', BURETA_CODE);
+
+  await db.runSql(`
+    UPDATE entity 
+    SET parent_id = '${BURETA_ID}'
```

Comment:
I believe that parent_id here should be set to Fiji to match other Fiji villages?

Reply:
I.e all villages are directly under Fiji for the historical reason mentioned here: https://github.com/beyondessential/tupaia-backlog/issues/1423#issuecomment-722877076
And set placed in a catchment for wish vis the entity_relation below.
This will keep them all consistent for when updated later.

Reply:
Thanks Nick for this catching! I see what you mean. I will change the parent_id to Fiji.